modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx

```diff
@@ -266,32 +254,54 @@ const SaveFDSFieldsModalContent = ({
 
 	const visibleFields = fields?.filter((field) => field.visible) ?? [];
 
-	return (
-		<>
-			<ClayModal.Header>
-				{Liferay.Language.get('add-fields')}
-			</ClayModal.Header>
-
-			<ClayModal.Body>
-				{fields === null ? (
-					<ClayLoadingIndicator />
-				) : (
-					<>
-						<ManagementToolbar.Container>
-							<ManagementToolbar.ItemList expand>
-								<ManagementToolbar.Item className="pr-2">
-									<ClayCheckbox
-										checked={isSelectAllChecked()}
-										indeterminate={isSelectAllIndeterminate()}
-										onChange={({target: {checked}}) =>
-											setFields(
-												fields.map((field) => ({
-													...field,
-													selected: checked,
-												}))
-											)
-										}
-									/>
+	const isChecked = useMemo(() => {
```

Comment:
This variable should be `isSelectAllChecked` , because there are other checkboxes in the scope. The same for `isIndeterminate`.